workers/unity/Assets/Playground/Scripts/Worker/ClientWorkerConnector.cs

```diff
@@ -15,13 +17,45 @@ public class ClientWorkerConnector : DefaultWorkerConnector
         private async void Start()
         {
             Application.targetFrameRate = 60;
-            await Connect(WorkerUtils.UnityClient, new ForwardingDispatcher()).ConfigureAwait(false);
-        }
 
-        protected override string SelectDeploymentName(DeploymentList deployments)
-        {
-            // This could be replaced with a splash screen asking to select a deployment or some other user-defined logic.
-            return deployments.Deployments[0].DeploymentName;
+            var connParams = new ConnectionParameters
+            {
+                WorkerType = WorkerUtils.UnityClient,
+                DefaultComponentVtable = new ComponentVtable(),
+                Network = new NetworkParameters
+                {
+                    UseExternalIp = UseExternalIp
```

Comment:
this will be overwritten when `resolve` is called, right?

Reply:
It can be